lib/oli/rendering/content/selection.ex

```diff
@@ -94,7 +94,7 @@ defmodule Oli.Rendering.Content.Selection do
         %{}
 
       ids ->
-        Oli.Publishing.DeliveryResolver.from_resource_id(section_slug, ids)
+        Oli.Publishing.AuthoringResolver.from_resource_id(section_slug, ids)
```

Comment:
I'm not sure if I'm understanding this change. The `DeliveryResolver` function expects a section slug while the `AuthoringResolver` one expects a project slug, so why are we switching functions here?